CHANGELOG.md

```diff
@@ -6,829 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Upcoming
 
-### 🔄 Changed
-
```

Comment:
I don't think we should remove the old stuff! Just add a new version :)

Reply:
it'll be interesting then, old CHANGELOG will continue (2.6.4, 2.7, 2.7.1....) but this changelog will jump from 2.6.2 -> 3.0.0

Reply:
All right with me 👍